### ISSUE TITLE
[GCC11] GEMDQMBase::MEStationInfo::nMinIdxChamber_ may be used undefined

### ISSUE THREAD

**Issue by iarspider** [cms-sw/cmssw#38516]
In gcc11 builds, the following warning is emitted:
```
>> Compiling  /data/cmsbld/jenkins/workspace/build-any-ib/w/tmp/BUILDROOT/89ea3e9c59449ee799ede0a2b9444490/opt/cmssw/el9_amd64_gcc11/cms/cmssw/CMSSW_12_5_X_2022-06-26-2300/src/DQM/GEM/src/GEMDQMEfficiencySourceBase.cc
/data/cmsbld/jenkins/workspace/build-any-ib/w/el9_amd64_gcc11/external/gcc/11.2.1-f478fee2760dbd22aaabb4e3a8fe1640/bin/c++ -c -DGNU_GCC -D_GNU_SOURCE -DEIGEN_DONT_PARALLELIZE -DTBB_USE_GLIBCXX_VERSION=110201 -DTBB_SUPPRESS_DEPRECATED_MESSAGES -DTBB_PREVIEW_RESUMABLE_TASKS=1 -DTBB_PREVIEW_TASK_GROUP_EXTENSIONS=1 -DBOOST_SPIRIT_THREADSAFE -DPHOENIX_THREADSAFE -DBOOST_MATH_DISABLE_STD_FPCLASSIFY -DBOOST_UUID_RANDOM_PROVIDER_FORCE_POSIX -D__STDC_LIMIT_MACROS -D__STDC_FORMAT_MACROS -DCMSSW_GIT_HASH='CMSSW_12_5_X_2022-06-26-2300' -DPROJECT_NAME='CMSSW' -DPROJECT_VERSION='CMSSW_12_5_X_2022-06-26-2300' -I/data/cmsbld/jenkins/workspace/build-any-ib/w/tmp/BUILDROOT/89ea3e9c59449ee799ede0a2b9444490/opt/cmssw/el9_amd64_gcc11/cms/cmssw/CMSSW_12_5_X_2022-06-26-2300/src -I/data/cmsbld/jenkins/workspace/build-any-ib/w/el9_amd64_gcc11/external/classlib/3.1.3-96b8a9d6b313cd5e41e38abfe614f509/include -I/data/cmsbld/jenkins/workspace/build-any-ib/w/el9_amd64_gcc11/external/pcre/8.43-e6b3d6f8e1424033990367f0ae94d8f3/include -isystem/data/cmsbld/jenkins/workspace/build-any-ib/w/el9_amd64_gcc11/external/boost/1.78.0-81e6e1bee645eae389961f9b71715fc1/include -I/data/cmsbld/jenkins/workspace/build-any-ib/w/el9_amd64_gcc11/external/bz2lib/1.0.6-a8795c5492b25410af2fdbfc34f86eab/include -isystem/data/cmsbld/jenkins/workspace/build-any-ib/w/el9_amd64_gcc11/external/clhep/2.4.5.1-863959129834eec6fab65756f04dfddf/include -I/data/cmsbld/jenkins/workspace/build-any-ib/w/el9_amd64_gcc11/external/gsl/2.6-a098c4550ea3678e68f90105e66b7865/include -I/data/cmsbld/jenkins/workspace/build-any-ib/w/el9_amd64_gcc11/external/libuuid/2.34-548af62999d8f41d39f61053ab3a9206/include -I/data/cmsbld/jenkins/workspace/build-any-ib/w/el9_amd64_gcc11/external/protobuf/3.15.1-1b81c2ae1e1d16f5253956ba30751417/include -isystem/data/cmsbld/jenkins/workspace/build-any-ib/w/el9_amd64_gcc11/lcg/root/6.24.07-9fdc8869fd6ec2752306673f06de5dd9/include -isystem/data/cmsbld/jenkins/workspace/build-any-ib/w/el9_amd64_gcc11/external/tbb/v2021.5.0-6242e961f4d35ec3f4a7fc9d422b635c/include -I/data/cmsbld/jenkins/workspace/build-any-ib/w/el9_amd64_gcc11/cms/vdt/0.4.3-bafe9c27f0b93dfeb9390c930c3c6f14/include -I/data/cmsbld/jenkins/workspace/build-any-ib/w/el9_amd64_gcc11/external/xz/5.2.5-4fdf584c0d81808d2831b2beffcb3bc7/include -I/data/cmsbld/jenkins/workspace/build-any-ib/w/el9_amd64_gcc11/external/zlib/1.2.11-31d2ceb65830b90a6cc7a5a90279f5d6/include -I/data/cmsbld/jenkins/workspace/build-any-ib/w/el9_amd64_gcc11/external/eigen/82dd3710dac619448f50331c1d6a35da673f764a-4aefcc7388143cab8a13000c7d39a794/include/eigen3 -I/data/cmsbld/jenkins/workspace/build-any-ib/w/el9_amd64_gcc11/external/fmt/8.0.1-69e27e17e6d62c775e8a26ac0fdbe1c9/include -I/data/cmsbld/jenkins/workspace/build-any-ib/w/el9_amd64_gcc11/external/md5/1.0.0-65713d22725e6dea36af6865a8b7982a/include -I/data/cmsbld/jenkins/workspace/build-any-ib/w/el9_amd64_gcc11/external/OpenBLAS/0.3.15-5e792517c6136595840da4cbd77980ac/include -I/data/cmsbld/jenkins/workspace/build-any-ib/w/el9_amd64_gcc11/external/tinyxml2/6.2.0-dff3d5cfeccbaf05917bf8c158421d93/include -O2 -pthread -pipe -Werror=main -Werror=pointer-arith -Werror=overlength-strings -Wno-vla -Werror=overflow -std=c++1z -ftree-vectorize -Wstrict-overflow -Werror=array-bounds -Werror=format-contains-nul -Werror=type-limits -fvisibility-inlines-hidden -fno-math-errno --param vect-max-version-for-alias-checks=50 -Xassembler --compress-debug-sections -msse3 -felide-constructors -fmessage-length=0 -Wall -Wno-non-template-friend -Wno-long-long -Wreturn-type -Wextra -Wpessimizing-move -Wclass-memaccess -Wno-cast-function-type -Wno-unused-but-set-parameter -Wno-ignored-qualifiers -Wno-deprecated-copy -Wno-unused-parameter -Wunused -Wparentheses -Wno-deprecated -Werror=return-type -Werror=missing-braces -Werror=unused-value -Werror=unused-label -Werror=address -Werror=format -Werror=sign-compare -Werror=write-strings -Werror=delete-non-virtual-dtor -Werror=strict-aliasing -Werror=narrowing -Werror=unused-but-set-variable -Werror=reorder -Werror=unused-variable -Werror=conversion-null -Werror=return-local-addr -Wnon-virtual-dtor -Werror=switch -fdiagnostics-show-option -Wno-unused-local-typedefs -Wno-attributes -Wno-psabi -Wno-error=unused-variable -DBOOST_DISABLE_ASSERTS  -fPIC  -MMD -MF tmp/el9_amd64_gcc11/src/DQM/GEM/src/DQMGEM/GEMDQMEfficiencySourceBase.cc.d /data/cmsbld/jenkins/workspace/build-any-ib/w/tmp/BUILDROOT/89ea3e9c59449ee799ede0a2b9444490/opt/cmssw/el9_amd64_gcc11/cms/cmssw/CMSSW_12_5_X_2022-06-26-2300/src/DQM/GEM/src/GEMDQMEfficiencySourceBase.cc -o tmp/el9_amd64_gcc11/src/DQM/GEM/src/DQMGEM/GEMDQMEfficiencySourceBase.cc.o
In file included from /data/cmsbld/jenkins/workspace/build-any-ib/w/tmp/BUILDROOT/89ea3e9c59449ee799ede0a2b9444490/opt/cmssw/el9_amd64_gcc11/cms/cmssw/CMSSW_12_5_X_2022-06-26-2300/src/DQM/GEM/src/GEMDQMBase.cc:1:
In member function 'GEMDQMBase::MEStationInfo& GEMDQMBase::MEStationInfo::operator=(GEMDQMBase::MEStationInfo&&)',
    inlined from 'int GEMDQMBase::loadChambers()' at /data/cmsbld/jenkins/workspace/build-any-ib/w/tmp/BUILDROOT/89ea3e9c59449ee799ede0a2b9444490/opt/cmssw/el9_amd64_gcc11/cms/cmssw/CMSSW_12_5_X_2022-06-26-2300/src/DQM/GEM/src/GEMDQMBase.cc:111:119:
  /data/cmsbld/jenkins/workspace/build-any-ib/w/tmp/BUILDROOT/89ea3e9c59449ee799ede0a2b9444490/opt/cmssw/el9_amd64_gcc11/cms/cmssw/CMSSW_12_5_X_2022-06-26-2300/src/DQM/GEM/interface/GEMDQMBase.h:462:9: warning: '*(long unsigned int*)((char*)&<unnamed> + offsetof(GEMDQMBase::MEStationInfo, GEMDQMBase::MEStationInfo::nMinIdxChamber_))' may be used uninitialized [-Wmaybe-uninitialized]
   462 |   class MEStationInfo {
      |         ^~~~~~~~~~~~~
/data/cmsbld/jenkins/workspace/build-any-ib/w/tmp/BUILDROOT/89ea3e9c59449ee799ede0a2b9444490/opt/cmssw/el9_amd64_gcc11/cms/cmssw/CMSSW_12_5_X_2022-06-26-2300/src/DQM/GEM/src/GEMDQMBase.cc: In member function 'int GEMDQMBase::loadChambers()':
/data/cmsbld/jenkins/workspace/build-any-ib/w/tmp/BUILDROOT/89ea3e9c59449ee799ede0a2b9444490/opt/cmssw/el9_amd64_gcc11/cms/cmssw/CMSSW_12_5_X_2022-06-26-2300/src/DQM/GEM/src/GEMDQMBase.cc:111:119: note: '<anonymous>' declared here
  111 |             MEStationInfo(region_number, station_number, layer_number, num_superchambers, num_etas, num_vfat, num_digi);
      |                                                                                                                       ^
```

(e.g. [link](https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/el9_amd64_gcc11/CMSSW_12_5_X_2022-06-26-2300/DQM/GEM)).

**Comment by iarspider** [comment]
assign dqm

**Comment by makortel** [comment]
FYI @cms-sw/gem-dpg-l2

**Comment by makortel** [comment]
Or, was this actually fixed already in #38517? @iarspider

**Comment by iarspider** [comment]
@makortel it was fixed with #38517
